rust/main/hyperlane-test/src/setup/relayer_server.rs

```diff
@@ -0,0 +1,29 @@
+use axum::Router;
+use prometheus::IntGaugeVec;
+use std::net::SocketAddr;
+
+pub fn dummy_metrics_and_label() -> (IntGaugeVec, String) {
```

Comment:
same here, the opqueue only exists in the relayer so it doesn't really make sense to move it out of that crate